packages/backend/package.json

```diff
@@ -20,7 +20,7 @@
     "@backstage/catalog-client": "^1.4.2",
     "@backstage/catalog-model": "^1.4.0",
     "@backstage/config": "^1.0.8",
-    "@backstage/integration": "^1.5.0",
+    "@backstage/integration": "^1.5.1",
```

Comment:
Can you revert upgrading `@backstage/integration`?

Reply:
Done